### DISCUSSION TITLE
Fehlernummer auslesbar?

### DISCUSSION THREAD

**Discussion by AndiMue** [tbnobody/OpenDTU#1373] [Q&A]
Ist die Fehlernummer ein DTU internes Signal oder kommt das vom WR, z.B. 1 für Inverter Start? Ich würde die Nummer gern auslesen und an meine Homematic schicken können, habe aber auch keinen Befehl dazu in der Übersicht gefunden.

**Comment by DPO99** [comment]
Kommt vom Wechselrichter. M.E. wird das derzeit nirgends reingeschrieben, wo es auch auslesbar ist.

EDIT
Per API gehts: https://github.com/tbnobody/OpenDTU/blob/master/docs/Web-API.md


$ curl --no-progress-meter http://192.168.10.10/api/eventlog/status?inv=11418186xxxx | jq
{
  "11418186xxxx": {
    "count": 4,
    "events": [
      {
        "message_id": 1,
        "message": "Inverter start",
        "start_time": 28028,
        "end_time": 28028
      },
      {
        "message_id": 209,
        "message": "PV-1: No input",
        "start_time": 28036,
        "end_time": 0
      },
      {
        "message_id": 2,
        "message": "DTU command failed",
        "start_time": 28092,
        "end_time": 28092
      },
      {
        "message_id": 207,
        "message": "MPPT-A: Input undervoltage",
        "start_time": 28336,
        "end_time": 0
      }
    ]
  }
}
